x-pack/test/detection_engine_api_integration/security_and_spaces/bundled_prebuilt_rules_package/prerelease_packages.ts

```diff
@@ -39,6 +40,9 @@ export default ({ getService }: FtrProviderContext): void => {
 
       await installPrebuiltRules(es, supertest);
 
+      // Refresh ES indices to avoid race conditions between write and reading of indeces
+      await es.indices.refresh({ index: ALL_SAVED_OBJECT_INDICES });
```

Comment:
I see `installPrebuiltRules` [has already](https://github.com/elastic/kibana/blob/main/x-pack/test/detection_engine_api_integration/utils/prebuilt_rules/install_prebuilt_rules.ts#L59) the same index refresh.